src/conduits/java/com/enderio/conduits/common/network/ConduitSavedData.java

```diff
@@ -25,13 +26,16 @@
 import net.minecraftforge.eventbus.api.SubscribeEvent;
 import net.minecraftforge.fml.common.Mod;
 
+import javax.annotation.Nullable;
```

Comment:
remove this unused annotation, also we use org.jetbrains.annotations.Nullable

Reply:
Fixed [be989a0](https://github.com/SleepyTrousers/EnderIO-Rewrite/pull/85/commits/be989a08650d0dfddd134d370b3e408df39f8f36)